src/views/collection-view/server.ts

```diff
@@ -128,8 +128,13 @@ export async function getCollectionPagePaths(): Promise<CollectionPagePath[]> {
 		 *
 		 * @TODO once we implement the `theme` query option, remove the theme filtering
 		 * https://app.asana.com/0/1201903760348480/1201932088801131/f
+		 *
+		 * @TODO ignoring `hcp` product slug for now until we know whether or not
+		 * we're using "hcp" or "cloud".
+		 * https://app.asana.com/0/1202097197789424/1202618936981037/f
 		 */
 		const shouldBuildCollectionPath =
+			productSlug !== 'hcp' &&
```

Comment:
🗒️ This needed to be added so that we don't build tutorials/collections routes for the `hcp` slug (which isn't a slug in Learn).